app/src/main/java/com/flexicharge/bolt/helpers/Validator.kt

```diff
@@ -49,6 +49,14 @@ class Validator {
                             else{
                                 field.error = null
                             }
+                        TextInputType.isTooLong ->
+                            if (s.length > 15){
+                                field.error = "Name can't be longer than 15 characters"
+                            }
+                        else{
+                            field.error = null
+                            }
+
```

Comment:
Indentation "error". The use of whitespaces is very inconsistent throughout the whole codebase. I will update the coding standard later this week. 
```suggestion
                         TextInputType.isTooLong ->
                            if (s.length > 15) {
                                field.error = "Name can't be longer than 15 characters"
                            }
                            else {
                                field.error = null
                            }
```